src/app/public/modules/skip-link/skip-link-host.component.ts

```diff
@@ -0,0 +1,47 @@
+// #region imports
+
+import {
+  ChangeDetectorRef,
+  ChangeDetectionStrategy,
+  Component
+} from '@angular/core';
+
+import {
+  SkySkipLink
+} from './skip-link';
+
+import {
+  SkySkipLinkAdapterService
+} from './skip-link-adapter.service';
+
+// #endregion
+
+@Component({
+  selector: 'sky-skip-link-host',
+  templateUrl: './skip-link-host.component.html',
+  styleUrls: ['./skip-link-host.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SkySkipLinkHostComponent {
+
+  public get links(): SkySkipLink[] {
+    return this._links;
+  }
+
+  public set links(value: SkySkipLink[]) {
+    this._links = value;
```

Comment:
I noticed if you try and pass a `SkySkipLink` to this with an undefined `elementRef`, there's no error until the user tries to tab into the hidden link. Should we alert the consumer if they try to pass it an `elementRef` that's not found?